samples/CommunityToolkit.Maui.Sample/Models/AvatarModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace CommunityToolkit.Maui.Sample.Models;
+
+public class AvatarModel
+{
+	public Color BackgroundColor { get; init; } = Colors.Black;
+	public Color BorderColor { get; init; } = Colors.White;
+	public Thickness BorderWidth { get; init; } = 0;
+	public CornerRadius CornerRadius { get; init; } = new(50, 50, 50, 50);
+	public string Description { get; init; } = string.Empty;
+	public FontAttributes FontAttributes { get; init; } = FontAttributes.None;
+	public double FontSize { get; init; } = 18;
+	public double HeightRequest { get; init; } = 64;
```

Comment:
Can we use SizeRequest so we ensure both Width and Height are set and have a valid value?

Reply:
Let's keep `HeightRequest` and `WidthRequest` separate because it is easier to then use their values for XAML Bindings